src/frontend/services/save-project-as.ts

```
@@ -0,0 +1,129 @@
+ * redirection.
+ *
+ * It is not retrieval-specific. Any project can be written somewhere else, and
+ * the retrieved case is simply the one where it is not optional.
+ */
+
+import type { PlatformCapabilities } from '../../middleware/shared/ports/platform-capabilities'
+import type { ProjectPort } from '../../middleware/shared/ports/project-port'
+import { openPLCStoreBase } from '../store'
+import { toast } from '../utils/toast'
+import { flushFlowWriteBacks } from '../store/slices/shared/flow-writeback'
+import { buildAllProjectFileContents, flushGlobalVariableListDrafts } from './save-actions'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)